client/state/automated-transfer/constants.js

```diff
@@ -1,7 +1,16 @@
 export const AUTOMATED_TRANSFER_STATUS = {
+	INQUIRING: 'inquiring',
 	START: 'start',
 	SETUP: 'setup',
 	LEAVING: 'leaving',
 	CONFLICTS: 'conflicts',
 	COMPLETE: 'complete',
 };
+
+export const eligibilityHolds = {
+	NO_WPCOM_NAMESERVERS: 'NO_WPCOM_NAMESERVERS',
+	NON_ADMIN_USER: 'NON_ADMIN_USER',
+	NOT_USING_CUSTOM_DOMAIN: 'NOT_USING_CUSTOM_DOMAIN',
+	SITE_GRAYLISTED: 'SITE_GRAYLISTED',
+	SITE_PRIVATE: 'SITE_PRIVATE',
+};
```

Comment:
@copons I didn't like using the SCREAMING_SNAKE_CASE for the object name. The object itself seemed to be more of a container of constants than a constant itself, so I was reserving SCREAMING_SNAKE_CASE for those things which we would ideally have as Symbols in a different programming language. Was planning on bringing this up with you.